arkouda/pdarrayclass.py

```diff
@@ -1949,7 +1949,8 @@ def argmin(pda: pdarray) -> np.int64:
     RuntimeError
         Raised if there's a server-side error thrown
     """
-    repMsg = generic_msg(cmd="reduction", args="{} {}".format("argmin", pda.name))
+    # using .name due to issues with Datetime
+    repMsg = generic_msg(cmd="reduction", args={"op": "argmin", "array": pda.name})
```

Comment:
sorry to add one more, I think this one should be good to go now too

Reply:
Good catch. Updated